### PR TITLE
💄 MT-891  Moved style out of the article-spacing mixin

### PR DESCRIPTION
**Pull request by miloszhygen**
the summary details needs less space on top and bottom.

### CODE REVIEW

File: src/scss/components/_legal-collection.scss [hunk 1]
```diff
@@ -4,12 +4,16 @@
 .legal-collection {
```
Comment (lha047) [inline src/scss/components/_legal-collection.scss:4]: Can you add a `LegalGuidanceCollection.stories.svelte` to display how the legal guidance collection should look like?

**Review by lha047 — CHANGES_REQUESTED** [review]
Code looks good :D 

Could you make a story for legal guidance collection?

File: src/scss/components/_legal-collection.scss (hunk 1 above)
Comment (miloszhygen) [reply src/scss/components/_legal-collection.scss:4]: I'll fix :)

**Review by lha047 — APPROVED** [review]
Looks good :D

**Pull request merged**